cmd/bolt/main.go

```diff
@@ -487,16 +500,24 @@ func (cmd *PageCommand) Run(args ...string) error {
 		}
 
 		// Print basic page info.
-		fmt.Fprintf(cmd.Stdout, "Page ID:    %d\n", p.id)
-		fmt.Fprintf(cmd.Stdout, "Page Type:  %s\n", p.Type())
-		fmt.Fprintf(cmd.Stdout, "Total Size: %d bytes\n", len(buf))
+		if options.itemKey == -1 && options.itemValue == -1 {
```

Comment:
judging by the special-casing and that `pages` is described as `print one or more pages in human readable format`, this might be better as a separate command `dump-item [-key|-value] PATH pageid idx` instead of overloading the pages command

Reply:
Humm, yeah. I had been thinking about splitting this out. I'll do it now.